vagrant/centos.sh

```diff
@@ -22,9 +22,9 @@ yum install -y epel-release
 
 yum -y groupinstall "Development tools"
 
-yum install -y git unzip gettext libxml2-devel libxslt-devel openssl-devel libffi-devel python-devel python-pip python-virtualenvwrapper rabbitmq-server
+yum install -y git unzip gettext libxml2-devel libxslt-devel openssl-devel libffi-devel python-devel python-pip python-virtualenvwrapper redis-server
```

Comment:
There is no redis-server in yum install.... Please refer to https://gist.github.com/nghuuphuoc/7801123 for a installation of redis